blog/content/post/2023-10-22-liquid-cooling.md

```diff
@@ -0,0 +1,147 @@
+pump is actually not in the CPU cooling block, but in the radiator itself!
+
+![Pump and radiator](/img/liquid_cooling_radiator.jpg)
+
+Remembering physics back from the school, if I placed the radiator with the
+pipes on the upper side, the air bubbles would not get into the pump right? To
+be sure I have searched for a more suitable tutorial, specific to my liquid
+cooling model and I found [Gear Seeker's
+guide](https://www.youtube.com/watch?v=ayE9X71SDeY) which explicitly says that
+this cooling should be mounted with the pipes upside. You may imagine what
+relief I have felt when I watched it. I took apart the cooling from my chassis
+right away and started mounting it with the pipes on the upper side. THanks to
+that the pipes would no longer need to go around my GPU (they would actually
+be hanging above the card with plenty of free space).
+
+![Liquid cooling mounted](/img/liquid_cooling_mounted.jpg)
+
+### Cooling efficiency
+
+I have no comparison of liquid cooling on a i5-12600K, so I can only share the
+results of i9-14900K. When idling the CPU stays at around 40 Celsius degrees.
+
+![Idle CPU temperature](/img/cpu_idle_temp.png)
+
+When playing a not-so-heavy game like League of Legends for example, the
+temperature varies from 60 to 80 degrees depending on the load.
+
+![CPU temperature under semi load](/img/cpu_semi_load_temp.png)
+
+This is where the fans are getting a bit noisy. I am considering adjusting the
+fan curves to reduce the loudness. Also the pump is generating its own
+constant noise, which is much more audible than fan+radiator solution.
+
+![CPU fan curve](/img/cpu_fan_curve.png)
+
+Liquid cooling should have a bit more heat capacity than fan+radiator solution
+and so the inertia of cooling will be different (there is no need to spin the
+fans to high speed right away when CPU crosses 80 degrees, because the liquid
+is still probably much colder than that). But that is a topic for another
+story and hopefully a blog post.
+
+## Summary
+
+Had I watched more tutorials I could save a few hours of time and nerves...
+Despite being an experienced engineer working with hardware everyday, I still
+underestimated the amount of information I need to get the best out of the
+products I have bought.
+
+Unlock the full potential of your hardware and secure your firmware with the
+experts at 3mdeb! If you're looking to boost your product's performance and
+protect it from potential security threats, our team is here to help.
+[Schedule a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
+or drop us an email at `contact<at>3mdeb<dot>com` to start unlocking the hidden
+benefits of your hardware. And if you want to stay up-to-date on all things
+firmware security and optimization, be sure to
+[sign up for our newsletter](https://newsletter.3mdeb.com/subscription/PW6XnCeK6).
+Don't let your hardware hold you back, work with 3mdeb to achieve more!
```

Comment:
I would need two things here:
- promotion of your Twitter account - if you like content like this feel free to flow me yada yada
- promotion of 3mdeb account in similar fashion

Reply:
https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590R151-R162